golang/lib/binding_constructors/binding_constructors.go

```diff
@@ -239,15 +235,17 @@ func NewExecCommandResponse(exitCode int32, logOutput []byte) *kurtosis_core_rpc
 //                              Wait For Endpoint Availability
 // ==============================================================================================
 func NewWaitForEndpointAvailabilityArgs(
-		serviceId string,
-		port uint32,
-		path string,
-		initialDelaySeconds uint32,
-		retries uint32,
-		retriesDelayMilliseconds uint32,
-		bodyText string) *kurtosis_core_rpc_api_bindings.WaitForEndpointAvailabilityArgs {
+	serviceId string,
+	httpMethod uint32,
+	port uint32,
+	path string,
```

Comment:
Same here and everywhere - the args-being-indented-an-extra-level is intentional, so that the args don't just look like functiona body code visually